src/com/griddynamics/cinema/MainCinema.java

```diff
@@ -0,0 +1,7 @@
+package com.griddynamics.cinema;
+
+public class MainCinema {
+        public static void main(String[] args) {
+        new Menu().cinemaMenu();
```

Comment:
What is Menu?

Reply:
Menu is class, implemented to give user the ability to change state or get information about current state of Cinema class object, by entering number of corresponding menu item in command line.

Reply:
call it "cli", would be easier to understand
https://en.wikipedia.org/wiki/Command-line_interface